lib/routes/routePUT.js

```diff
@@ -90,6 +90,14 @@ export default function routePUT(request, response, log, statsClient) {
                     return routesUtils.responseNoBody(err, null, response, 200,
                         log);
                 });
+            } else if (request.query.cors !== undefined) {
+                api.callApiMethod('bucketPutCors', request, log, err => {
+                    statsReport500(err, statsClient);
+                    // TODO: pushMetrics(err, log, utapi, 'bucketPutCors',
```

Comment:
actually this should be in the api now and I believe because of the refactor, it can be added without first updating utapi